keyboards/yugo_m/model_m_101/model_m_101.c

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2020 Nidzo Tomic <tomicn8@hotmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "model_m_101.h"
+
+void matrix_init_kb(void) {
+    // put your keyboard start-up code here
+    // runs once when the firmware starts up
+    matrix_init_user();
+    led_init_ports();
+}
+
+void led_init_ports() {
+    // Set our LED pins as output
+    palSetPadMode(GPIOA, 2, PAL_MODE_OUTPUT_PUSHPULL); // NumLock LED
+    palClearPad(GPIOA, 2);
+    palSetPadMode(GPIOA, 1, PAL_MODE_OUTPUT_PUSHPULL); // CapsLock LED
+    palClearPad(GPIOA, 1);
+    palSetPadMode(GPIOA, 0, PAL_MODE_OUTPUT_PUSHPULL); // ScrollLock LED
+    palClearPad(GPIOA, 0);
+}
+
+void led_set_kb(uint8_t usb_led) {
```

Comment:
Needs to be updated to <https://docs.qmk.fm/#/custom_quantum_functions?id=example-led_update_kb-implementation>

Reply:
Also done. Thank you for help!